### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by s0me1code** [khorshuheng/appflowy-integration#1]
I am interested, but no rust background ✌️. and I have no idea how to help, but I am interested

**Comment by s0me1code** [comment]
Just to be clear, you trying to integrate appflowy with github right?

**Comment by khorshuheng** [comment]
No, not for GitHub specifically. This is more for hosting examples of automating certain tasks with API, for example, create an event on Google when an event is created on AppFlowy calendar. There are still more endpoints required to be developed on AppFlowy Cloud for this to be possible.

**Comment by khorshuheng** [comment]
That being said, we are also working on Zapier integration, so I would expect most users to use that instead of using the API directly, which is what I plan to show case in this repo.

**Comment by s0me1code** [comment]
Sorry for the late response, I just saw the message.

Wow looks incredible,  I do not know which one to thank you for Zapier or the Appflowy Integration.

**Comment by s0me1code** [comment]
But wait a sec, this puts me under the mercy of Zapier free plan. 0_0

**Comment by s0me1code** [comment]
Yes by the way still open to help in the zaper thing too, and I will make sure to response faster next time ^^.

**Comment by khorshuheng** [comment]
Actually, if you check the code base in this repo, you will see that this is not the code base for Zapier integration. It is example for using the REST API directly.
